src/components/ProductList/index.jsx

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import ItemList from 'components/ItemList';
-
+import { getProductStatus, sortProducts } from 'components/Utils/helpers';
 function ProductList({ products }) {
```

Comment:
Well, **_only a suggestion_**, it will be interesting to pass to `ProductList` component some list **_sorted_** before 😀 , not inside the same component, because it's possible in the future you need to show a Product List but **_not sorted_**.

Reply:
If you don't want to show it neat in the future, you just have to remove that function and that's it.